added two titles with  corresponding links and location of tools (#5833)
MicrosoftDocs/windows-itpro-docs: windows/client-management/administrative-tools-in-windows-10.md

![Screenshot of folder of admin tools](images/admin-tools-folder.png)

> These tools were included in previous versions of Windows and the associated documentation for each tool should help you use these tools in Windows 10. The following list links to documentation for each tool.Below tools are located under C:\Windows\Sytem32\

Reviewer: ```suggestion
These tools were included in previous versions of Windows and the associated documentation for each tool should help you use these tools in Windows 10. The following list links to documentation for each tool. The tools are located within the folder C:\Windows\System32\ or its subfolders.
```
This is of course my interpretation, although I have not researched the exact location of each of the 19 tools in the list. I recommend inviting @JohanFreelancer9 to review this PR content and suggest any improvements to it.

> These tools were included in previous versions of Windows and the associated documentation for each tool should help you use these tools in Windows 10. The following list links to documentation for each tool. The tools are located within the folder C:\Windows\System32\ or its subfolders.

Reviewer 2: ```suggestion
These tools were included in previous versions of Windows and the associated documentation for each tool should help you use these tools in Windows 10. The following list provides links to documentation for each tool. The tools are located within the folder C:\Windows\System32\ or its subfolders.
```

Reviewer 3: @JohanFreelancer9 Is the folder path not wrapped around Markdown backticks?

-   [ODBC Data Sources]( https://go.microsoft.com/fwlink/p/?LinkId=708494)
-   [Performance Monitor](https://go.microsoft.com/fwlink/p/?LinkId=708495)
-   [Print Management](https://go.microsoft.com/fwlink/p/?LinkId=708496)

> -   [Recovery Drive](https://support.microsoft.com/en-us/help/4026852/windows-create-a-recovery-drive)

Reviewer 2: ```suggestion
-   [Recovery Drive](https://support.microsoft.com/help/4026852/windows-create-a-recovery-drive)
```

> -   [Registry Editor](https://docs.microsoft.com/en-us/windows/win32/sysinfo/registry)

Reviewer 2: ```suggestion
-   [Registry Editor](https://docs.microsoft.com/windows/win32/sysinfo/registry)
```
